db/migrate/20170710142023_add_covering_letter_type_to_appointment_summaries.rb

```diff
@@ -0,0 +1,5 @@
+class AddCoveringLetterTypeToAppointmentSummaries < ActiveRecord::Migration[5.1]
+  def change
+    add_column :appointment_summaries, :covering_letter_type, :string
```

Comment:
Cool. I allowed null to accommodate the existing records but if empty string is preferred then 👍 , will change.